pages/1.13/cli/autocompletion/index.md

```diff
@@ -7,29 +7,27 @@ excerpt: Enabling autocompletion for the CLI
 enterprise: false
 ---
 
-Autocompletion for the CLI is supported for `bash` and `zsh`.
-It was released with the 1.12 release so the CLI must be of version 0.7.x or greater. Please see https://downloads.dcos.io/cli/index.html for a correct version. \
-You will also need to have the `dcos` executable in your `$PATH`.
+Autocompletion for the CLI is supported for `bash` and `zsh`. You will need to have the `dcos` executable in your `$PATH` and use it with a cluster running on DC/OS 1.12 or later.
```

Comment:
This is a very rough entry. Can we say something like: DC/OS CLI 0.7 (which comes with DC/OS 1.13) now includes auto completion, Auto completion makes it easy to .....etc. For example <provide example workflow>